config/maintainers.json

```diff
@@ -20,16 +20,6 @@
       "link_url": null,
       "avatar_url": null
     },
-    {
```

Comment:
If you would like to remain marked as an alumnus, you'll need to add the following in place of your old entry:
```
{
  "github_username": "N-Parsons",
  "alumnus": true
},
```